azurerm/internal/services/attestation/tests/attestation_data_source_test.go

```diff
@@ -0,0 +1,40 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+)
+
+func TestAccDataSourceAzureRMAttestation_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_attestation", "test")
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMAttestationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAttestation_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMAttestationExists(data.ResourceName),
+					resource.TestCheckResourceAttrSet(data.ResourceName, "name"),
+					resource.TestCheckResourceAttrSet(data.ResourceName, "resource_group_name"),
```

Comment:
there's not much point checking these fields since they're required to look it up (so if they're missing, the Exists check will fail) - either we can remove these or set & check tags instead?

Reply:
Fixed